cartridges/int_bolt_core/cartridge/scripts/utils/oauthUtils.js

```diff
@@ -156,7 +156,7 @@ function createPlatformAccount(externalProfile, orderId, orderToken) {
         // save custom fields, configured in Bolt admin dash
         saveCustomFields(externalProfile, authenticatedCustomerProfile);
 
-        // set order to the customer if the account is created during checkout.
+        // optional: set order to the new created account if shopper creates account during checkout
```

Comment:
I think order reassign could be optional for merchants since it's all about SFCC order behavior. So that merchants can also make the call to overwrite `Order-Confirm` or not.